config/crd/kustomization.yaml

```diff
@@ -13,13 +15,17 @@ patchesStrategicMerge:
 - patches/webhook_in_ytsaurus.yaml
 - patches/webhook_in_spyts.yaml
 - patches/webhook_in_chyts.yaml
+#- patches/webhook_in_remoteytsauruses.yaml
```

Comment:
Why are these commented?

Reply:
Not sure, it is what kubebulder generator does. I suppose if we want to add webhooks for remote ytsaurus or remotenodes CRDs we will uncomment this.